index.html

```diff
@@ -6254,6 +6257,32 @@ <h2>Acknowledgements</h2>
 Christopher Allen, Heather Vescent, and Wayne Chang.
     </p>
 
+    <p>
+The Working Group thanks the following individuals for their contributions to
+this specification (in alphabetical order): Denis Ah-Kang, Nacho Alamillo,
+Christopher Allen, Joe Andrieu, Antonio, Phil Archer, George Aristy, Baha, Juan
+Benet, BigBlueHat, Dan Bolser, Chris Boscolo, Pelle Braendgaard, Daniel Buchner,
+Daniel Burnett, By_caballero, Tim Cappalli, Melvin Carvalho, David Chadwick,
+Wayne Chang, Sam Curren, Hai Dang, Tim Daubenschütz, Oskar van Deventer, Kim
+Hamilton Duffy, Arnaud Durand, Ken Ebert, Veikko Eeva, Carson Farmer, Nikos
+Fotiou, Gabe, Gayan, Ryan Grant, Peter Grassberger, Adrian Gropper, Amy Guy,
+Daniel Hardman, Kyle Den Hartog, Philippe Le Hegaret, Ivan Herman, Michael
+Herman, Alen Horvat, Dave Huseby, Marcel Jackisch, Mike Jones, Andrew Jones, Tom
+Jones, jonnycrunch, Ted Thibodeau Jr, Gregg Kellogg, Michael Klein, Paul
+Knowles, David I. Lehn, Charles Lehner, Michael Lodder, Dave Longley, Tobias
+Looker, Wolf McNally, Robert Mitwicki, Mircea Nistor, Grant Noble, Mark
+Nottingham, Darrell O'Donnell, Vinod Panicker, Dirk Porsche, Praveen, Mike
+Prorock, Drummond Reed, Julian Reschke, Yancy Ribbens, Justin Richer, Rieks,
+Mikeal Rogers, Evstifeev Roman, Troy Ronda, Leonard Rosenthol, Michael Ruminer,
+Markus Sabadello, Cihan Saglam, Samu, Rob Sanderson, Wendy Seltzer, Mehran
+Shakeri, Jaehoon (Ace) Shim, Samuel Smith, James M Snell, SondreB, Manu Sporny,
+Orie Steele, Shigeya Suzuki, Sammotic Switchyarn, Oliver Terbu, Joel
+Thorstensson, Tralcan, Henry Tsai, Rod Vagg, Mike Varley, Kaliya "Identity
+Woman" Young, Eric Welton, Fuqiao Xue, @Yue, Dmitri Zagidulin, Brent Zundel,
+cabo, ewagner70, gimly-jack, gjgd, kdenhartog-sybil1, ktobich, mooreT1881, oare,
+pukkamustard, rknobloch, ssstolk, tahpot, and zhanb.
```

Comment:
I started by correcting my alphabetical placement (sorting on Thibodeau, not Jr); then I noticed there were several people following Zundel who should have been sorted on their single-name-as-family-name; then realized that several of these were github handles which might be better kept as `@`names like `@Yue`, but maybe that's not a github handle, so....

It seems the only safe alpha-sort for these names would be by treating each name as a string -- no special treatment for given vs family vs github vs other-nickname vs....

Or maybe there's [no safe name sort at all](https://shinesolutions.com/2018/01/08/falsehoods-programmers-believe-about-names-with-examples/).

Reply:
... and if we add `@` to the beginning of all the github handles, they should sort first... which would be weird, 'cause who is `oare` and `mooreT1881` anyway and why are they at the front of the list?

I sorted by last word in name and then github handles in alphanum order (as you highlight, there is no really correct order). I didn't decloak people that didn't list their full name in their Github handle, unless they came to a public WG meeting... but maybe I shouldn't have done that... and so on, and so on for 30 minutes, @TallTed ... the result is the list above, and who knows if it's good enough. :)

... but, I did miss placing your name in the right place, so please feel free to do that (or I'll try to remember in my next pass). I also don't have a very strong opinion on sort order for single word names vs. multi word names... it's wrong any way you do it. :)

Reply:
In my experience, single-word names are usually best treated as "floaters" that go in the column chosen for the overall sort (that is -- treat them as "first" if sorting by "first"; treat them as "last" if sorting by "last").

As to the github handles, I'd keep the `@`, and include a note saying that's what those are.  For sorting, I'd treat the handles as single-word names, and sort them by the handle minus the `@`.

(In my perfect world, I'd also include known github handles, maybe "Ted (@TallTed) Thibodeau Jr" or similar, with the overall sorting still by last name.)

fwiw, as you may also have discovered, I've found a spreadsheet tends to be the best tool for the bulk of this sort of processing (of which I've done a fair amount, over 30 years of volunteer and professional publishing efforts).  You can easily have an extra "sort" column alongside those for first, last, github, other, etc.; use formula(s) to build the "presentation" name; and then export that last column to a plain text doc for the final fluff of changing line breaks to comma+spaces.

Or just keep what you've got, modulo relocating my name.  ;-)

Reply:
I added the `@` to all the Github names and sorted those into the alphabetical list (sorting them as if they were a last name), which is what I think you were asking for.

Implemented in ae5c53ca20e35ca81f2471515f423567cb89575c.

Not your perfect world where everyone has their Github handle by their name... it may make the list less readable AND will make the W3C link checker take *forever* AND I'm a bit loathe to point upwards of 50+ links to Github from a W3C spec AND I'd have to rewrite/debug some code instead of focusing on getting this Proposed Rec out... but getting 3 out of 4 editorial requests ain't bad. :)